Dockerfile

```diff
@@ -33,9 +33,11 @@ ENV JAVA_HOME /usr/lib/jvm/default-java
 
 ADD resources /tmp/resources
 
-#Add JAI and ImageIO for improved performance.
+# Optionally add JAI and ImageIO for improved performance.
 WORKDIR /tmp
-RUN wget http://download.java.net/media/jai/builds/release/1_1_3/jai-1_1_3-lib-linux-amd64.tar.gz && \
+ARG JAI_IMAGEIO=true
+RUN if [ "$JAI_IMAGEIO" = true ]; then \
```

Comment:
Native JAI is not under active development and will eventually be replaced by JAI-Ext that includes support for ROI and Nodata. For various ImageIO readers/writers alternative implementations have been developed.
Is nice to have the option in case you run into issues or want to test performace with and without these components.